gravitee-apim-console-webui/src/portal/navigation/portal-navigation.service.ts

```diff
@@ -53,6 +53,13 @@ export class PortalNavigationService {
         icon: 'gio:top-bar',
       });
     }
+    if (this.permissionService.hasAnyMatching(['environment-settings-r', 'environment-settings-u'])) {
```

Comment:
Combine menu items that check the same permissions (aka Top Bar + Catalog)

Reply:
Done